packages/commonwealth/server.ts

```diff
@@ -91,12 +90,24 @@ const start = async () => {
 
       // checking the DYNO env var ensures this only runs on one dyno
       if (config.APP_ENV === 'production' && process.env.DYNO === 'web.1') {
+        const { dispatchSDKPublishWorkflow } = await import(
+          './server/util/dispatchSDKPublishWorkflow'
+        );
         dispatchSDKPublishWorkflow().catch((e) =>
           log.error(
             `Failed to dispatch publishing workflow ${JSON.stringify(e)}`,
           ),
         );
       }
+
+      // bootstrap bindings when in dev mode and DEV_MODULITH is true
+      if (config.NODE_ENV === 'development' && config.DEV_MODULITH) {
```

Comment:
Maybe `DEV_MODULITH` should default to true? I think most engineers will want this on by default and we will only run these separately when we are debugging the services separately.

Reply:
Actually, never mind. Probably not a good idea since this requires RabbitMQ to be running...